bokehjs/gulp/tasks/scripts.coffee

```diff
@@ -17,8 +17,9 @@ gulp.task "scripts:build", ->
     entries: ['./src/coffee/main.coffee']
     extensions: [".coffee", ".eco"]
     debug: true
-
+  
   browserify opts
+    .transform("browserify-css", {'global': true, 'minify':true})
```

Comment:
You shouldn't need the global transform anymore, provided you are using the most recent releases of phosphor. Each repo now declares its own required transforms in `package.json`.